ferry/includes/transform_import.py

```diff
@@ -643,11 +639,16 @@ def import_evaluations(
 
     Parameters
     ----------
-    evaluation_narratives: DataFrame of narratives
-    evaluation_ratings: DataFrame of ratings
-    evaluation_statistics: DataFrame of statistics
-    evaluation_questions: DataFrame of questions
-    listings: listings DataFrame from import_courses
+    evaluation_narratives: Pandas DataFrame
```

Comment:
We could use python3's type hints syntax, which simplifies the docstring and will help when we eventually start using mypy
```python
def import_evaluations(
    evaluation_narratives: pd.DataFrame,
    evaluation_ratings: pd.DataFrame,
    evaluation_statistics: pd.DataFrame,
    evaluation_questions: pd.DataFrame,
    listings: pd.DataFrame,
):
```

Reply:
Cool, I'll add this in for everything. Didn't realize we could use Pandas types!